docs/source/api/deliveryservice_stats.rst

```diff
@@ -26,7 +26,11 @@
 Retrieves time-aggregated statistics on a specific :term:`Delivery Service`.
 
 :Auth. Required: Yes
-:Roles Required: None
+:Roles Required: None\ [#tenancy]_
```

Comment:
shouldn't this say "Read-Only and above" or however we're indicating "this role or anything greater"

Reply:
"None" is standard for "read-only" or above, because that's how the old docs did it. It can be changed - and that would be more correct - but that should be done globally.